pkg/services/validations/service.go

```diff
@@ -2,11 +2,13 @@ package validations
 
 import (
 	"net/http"
+
+	"github.com/grafana/grafana/pkg/services/datasources"
 )
 
 type PluginRequestValidator interface {
```

Comment:
Maybe now is a good opportunity to update the name here (and comment docs below). Maybe `DataSourceRequestValidator`?